src/readme/rules.ts

```diff
@@ -360,12 +360,26 @@ const rules: IRule[] = [
   },
   {
     available: true,
+    eslintRule: 'no-unsafe-finally',
+    tslintRule: 'no-unsafe-finally',
+    category: 'Possible Errors',
+    description: 'disallow control flow statements in finally blocks (recommended)',
+    eslintUrl: 'http://eslint.org/docs/rules/no-unsafe-finally',
+    tslintUrl: 'https://palantir.github.io/tslint/rules/no-unsafe-finally',
+    provider: 'native',
+    usage: `~~~json
```

Comment:
remove the usage (was copied from the `use-isnan` rule) since the usage is stated on the rule docs.